### PR TITLE
Close quotes on `kubernetes_context` values in `gcp-modular`

### PR DESCRIPTION
**Pull request by cameronraysmith**
Failure to close quotations in the `stack_file` `kubernetes_context` values leads to errors like

```sh
Importing stack 'mlops' into ZenML...
...
ParserError: while parsing a flow mapping
  in "<unicode string>", line 27, column 24:
            configuration: {"kubernetes_context": "gke_zenm ... 
                           ^
expected ',' or '}', but got '<scalar>'
  in "<unicode string>", line 27, column 75:
     ... t": "gke_zenml-cluster-6i6x, "synchronous": True}
                                         ^
```

when running commands such as

```sh
mlstacks deploy -f stack.yaml -d
zenml stack deploy -p gcp -n mlops -r us-east4 -f stack.yaml -d
```

in zenml `v0.44.2` and mlstacks `v0.7.5`.

### CODE REVIEW

**Comment by strickvl** [comment]
Thanks for catching this @cameronraysmith! We take PRs on the `develop` branch, so could you possibly rebase this PR / branch on develop instead of making it point to `main`? https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop has instructions in case you need / want that (taken from the sister core `zenml` repository).

**Review by strickvl — APPROVED** [review]
LGTM, pending tests passing. Thanks for the contribution!

**Comment by cameronraysmith** [comment]
Let me know if it needs to be rebased again.

**Comment by strickvl** [comment]
@cameronraysmith yeah actually not sure what the issue is... the error message seems to be unrelated to the changes + develop branch itself doesn't have issues...

**Comment by cameronraysmith** [comment]
What is the error message? It's rebased on develop up to

```sh
1b606fe (HEAD -> fix-kubernetes-context-quotation, origin/fix-kubernetes-context-quotation) fix: close quotes on kubernetes_context values
71b97b5 (upstream/develop, develop) Add `skypilot` support for AWS and GCP (#99)
5069e56 update PR template to discourage PRs on main (#101)
64ce5c9 Add remote state deployment + use by default (#97)
```

**Comment by strickvl** [comment]
I see what's going on -- [here](https://github.com/zenml-io/mlstacks/actions/runs/6314211920/job/17144127638?pr=100). It's not passing the internal secrets down since you're contributing from a forked branch, so that's why the tests don't work. In any case this is a tiny PR with only fixes, no breaking changes, so I'll merge it into `develop` once the rest of the tests pass. Thanks again!

**Comment by cameronraysmith** [comment]
These tests failed in your #99 here https://github.com/zenml-io/mlstacks/actions/runs/6310939091 so they aren't going to pass after rebasing, but since you already merged that one there shouldn't be an issue with this one.

**Review by strickvl — APPROVED** [review]

**Comment by cameronraysmith** [comment]
Thank you @strickvl . Apologies for the confusion!

**Pull request merged**
